### DISCUSSION TITLE
Where is line-wrap?

### DISCUSSION THREAD

**Discussion by aykaramba** [lem-project/lem#826] [General]
Hi.

I have been reading this page: https://github.com/lem-project/lem/releases

For [v1.9.2] it says: Rename editor variable truncate-lines to line-wrap.

When I M-x line-* I don't see line-wrap as an option to autocomplete.  Does this still exist?  Is there an alternate options that I don't understand?

Thanks.

**Comment by cxxxr** [comment]
`line-wrap` is an editor variable, not a command.
It can be set up in the following way.
```
(in-package :lem-user)
(setf (variable-value 'line-wrap :global) nil)
```

It is true that this might be better to have a command.
Thanks for the feedback.

**Comment by aykaramba** [comment]
Great!  That works perfectly.  There is a lot to learn, this is awesome, I appreciate the quick response.

**Comment by aykaramba** [comment]
Hi everyone.  Just re-opening this thread.  Is there a feature in Lem that would the equivalent of visual-line-mode for human readable plain text editing?

I did a grep on the source files but only "line-wrap" shows up and I don't see anything in the M-x menu that might work like visual-line-mode.  I am doing a lot more plain text editing these days and sometimes that might be a nice feature to have.

As an aside, there is an amazing amount of modes in Lem and a huge number of features.  Very fun to use.

**Reply by cxxxr** [reply]
Thanks for the feedback.
I'd like that feature too, but haven't gotten around to it.

**Reply by aykaramba** [reply]
Okay no problem!  Thanks!
